lib/handleResponse.js

```diff
@@ -2,10 +2,10 @@ const handleResponse = (error, response, body, callback) => {
   if (!error && response.statusCode == 200) {
     callback(body);
   } else {
-    console.log('Error');
-    if (error) console.error(error);
-    if (response) console.error(`HTTP Status code: ${response.statusCode}`);
-    if (body) console.log(body);
+    logger.Error('Error');
+    if (error) logger.Error(error);
+    if (response) logger.Error(`HTTP Status code: ${response.statusCode}`);
```

Comment:
It looks weird when we log something thats not an error in logger.error - maybe some more abstract names?

Bootstrap got out of it using `danger`, `info` etc. classes. 

** https://getbootstrap.com/docs/4.0/components/alerts/

Reply:
since it's an error handling block I will display all as Error. @pavelloz makes sense?